icepyx/core/spatial.py

```diff
@@ -759,10 +759,10 @@ def fmt_for_CMR(self) -> str:
 
         """
         # CMR keywords: ['bounding_box', 'polygon']
-        if self._ext_type == "bounding_box":
-            cmr_extent = ",".join(map(str, self._spatial_ext))
+        if self.extent_type == "bounding_box":
+            return ",".join(map(str, self._spatial_ext))
 
-        elif self._ext_type == "polygon":
+        elif self.extent_type == "polygon":
```

Comment:
Silly question: isn't this more computationally expensive since it adds another "step" to look at the property?

Reply:
I don't believe so - at least not in any significant way. `self.extent_type` just returns `self._ext_type`. I think I did this because it seemed more clear to me. Happy to change it back though!